### ISSUE TITLE
Unable to resolve module rpc-websockets/dist/lib/client 

### ISSUE THREAD

**Issue by connoremma** [wormhole-foundation/wormhole-sdk-ts#636]
### TLDR: The project does not build on react-native

* I'm using react-native version 0.72 
* Enabled unstable_enablePackageExports: true per [this issue](https://github.com/wormhole-foundation/wormhole-sdk-ts/issues/541) (the TS config change did not work)
* Now I'm just getting more import related errors from subsequent imports like this 
![image](https://github.com/user-attachments/assets/b468f36d-4430-4475-b540-a7fcd3533355)

My use case is that I want to CCTP from Solana to Evm (and vise versa) 

The examples to do so also don't work/are out of date and can't be replicated without copying in a bunch of (outdated) interfaces and other incompatible example code 

Any more direct guidance would be appreciated – I already have the solana web3 js connection/keypairs needed to build and broadcast the transaction

**Comment by kev1n-peters** [comment]
Should be fixed in 0.10.1 https://github.com/wormhole-foundation/wormhole-sdk-ts/pull/667

**Comment by kev1n-peters** [comment]
Closing this issue since it should be resolved
